console/src/components/UIKit/atoms/Tooltip/Tooltip.ts

```diff
@@ -4,6 +4,5 @@ import { space } from 'styled-system';
 export const StyledTooltip = styled.span`
   display: inline-block;
   position: relative;
-
   ${space}
```

Comment:
We should provide typings for `StyledTooltip` which are:
* SpaceProps (imported from `styled-system`)
* HTML span element attributes

```ts
import styled from 'styled-components';
import { space, SpaceProps } from 'styled-system';

export interface StyledTooltipProps
  extends SpaceProps,
    React.ComponentPropsWithRef<'span'> {}

export const StyledTooltip = styled.span<SpaceProps>`
  display: inline-block;
  position: relative;

  ${space}
`;

```